src/neural/network_blas.cc

```diff
@@ -220,23 +223,51 @@ class BlasNetwork : public Network {
       Transforms::InvertBatchNormStddev(conv2.bn_stddivs);
     }
 
-    Transforms::OffsetBatchNormMeans(weights_.policy.bn_means, weights_.policy.biases);
+    Transforms::OffsetBatchNormMeans(weights_.policy.bn_means,
+                                     weights_.policy.biases);
     Transforms::InvertBatchNormStddev(weights_.policy.bn_stddivs);
 
-    Transforms::OffsetBatchNormMeans(weights_.value.bn_means, weights_.value.biases);
+    Transforms::OffsetBatchNormMeans(weights_.value.bn_means,
+                                     weights_.value.biases);
     Transforms::InvertBatchNormStddev(weights_.value.bn_stddivs);
 
 #ifdef USE_OPENBLAS
-// openblas_set_num_threads(1);
-// printf("BLAS Core: %s\n", openblas_get_corename());
+    int num_procs = openblas_get_num_procs();
+    blas_cores = std::min(num_procs, blas_cores);
+    openblas_set_num_threads(blas_cores);
+    if (verbose) {
+      const char* core_name = openblas_get_corename();
+      const char* config = openblas_get_config();
+      fprintf(stderr, "BLAS vendor: OpenBlas.\n");
```

Comment:
It's still better to do it using `std::cerr`, but if I'm find with doing that myself later.

Reply:
I said earlier that it was find to do, but after the merge of the 2 PRs.
Otherwise bad things (conflicts) may happens.